.github/workflows/linux.yml

```diff
@@ -1,6 +1,27 @@
 name: CI - Linux via APT
 
-on: [push,pull_request]
+on:
+  push:
+    paths-ignore:
```

Comment:
maybe we could add `.pre-commit-config.yaml` ?

Reply:
You're right. In fact, we don't have any CI job that check the pre-commit. I will add it.